src/diffbot.js

```diff
@@ -261,10 +265,12 @@ class Diffbot {
           throw new Error('missing name');
         else if (!options.seeds || !options.seeds.length)
           throw new Error('missing seeds');
-
+        let websites = options.seeds.split(' ');
```

Comment:
`seeds` should be passed a string array. If you need to split a string, it should be done in your app code, not the library.